library/core/src/primitive_docs.rs

```diff
@@ -268,6 +268,52 @@ mod prim_bool {}
 /// [`Debug`]: fmt::Debug
 /// [`default()`]: Default::default
 ///
+/// # Never type fallback
+///
+/// When the compiler sees a value of type `!` it implicitly inserts a coercion (if possible)
```

Comment:
“Possible” is vague. When is it possible? Any coercion site? If so, this should link to the concept of coercion sites.

Reply:
For reference, https://doc.rust-lang.org/reference/type-coercions.html#coercion-sites

Reply:
Yeah, good call. So what I have in mind then is

```suggestion
When the compiler sees a value of type `!` in a [coercion site](https://doc.rust-lang.org/reference/type-coercions.html#coercion-sites) it implicitly inserts a coercion
```

Reply:
Sounds good. Needs a comma after "coercion site".